packages/payment-processor/src/payment/near-conversion.ts

```diff
@@ -37,13 +38,7 @@ export async function payNearConversionRequest(
     throw new UnsupportedCurrencyError(request.currencyInfo);
   }
 
-  if (!paymentReference) {
-    throw new Error('Cannot pay without a paymentReference');
-  }
-
-  if (!network || !NearChains.isChainSupported(network)) {
-    throw new Error('Should be a Near network');
-  }
```

Comment:
Is the removal of this logic intentional?

Reply:
Nevermind. Removed because redundant with `NearChains.assertChainSupported(network)`

Reply:
Yes, should have made this more explicit in the PR indeed.